From a59eecaeecbd349059a2beccbdbadb2394e1994c Mon Sep 17 00:00:00 2001
From: "Suresh, Prashanth" <prashanth.suresh@intel.com>
Date: Wed, 29 Dec 2021 11:06:39 +0530
Subject: [PATCH] Disable trusty for CVTE.

Disable trusty as it is not needed for CVTE project currently.

Tracked-On: OAM-100495
Signed-off-by: Suresh, Prashanth <prashanth.suresh@intel.com>

diff --git a/caas/mixins.spec b/caas/mixins.spec
index 48518d0..db3bc6a 100755
--- a/caas/mixins.spec
+++ b/caas/mixins.spec
@@ -53,7 +53,7 @@ lights: true
 power: true(power_throttle=true)
 debug-usb-config: true(source_dev=dvcith-0-msc0)
 intel_prop: true
-trusty: true(ref_target=celadon_64)
+trusty: false
 memtrack: true
 avb: true
 avx: auto
-- 
2.34.1

